presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -25,11 +25,8 @@ connection properties as appropriate for your setup:
 Multiple MySQL Servers
 ^^^^^^^^^^^^^^^^^^^^^^
 
-You can have as many catalogs as you need, so if you have additional
```

Comment:
Let's remove this commit for now. We have the same information for other connectors like Kafka, MongoDB, etc., that all need to be updated.

I just re-read the `catalog-properties` section, and it's not as clear as it could be. I'd rather err on the side of overly-redundant in the documentation.